Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/test/ui/CertificatesViewModel.kt

```diff
@@ -42,6 +45,18 @@ class CertificatesViewModel @AssistedInject constructor(
 
     val events = SingleLiveEvent<CertificatesFragmentEvents>()
 
+    val markNewCertsAsSeen = testCertificateRepository.certificates
+        .onEach { wrappers ->
+            wrappers
+                .filter { !it.seenByUser && !it.isCertificateRetrievalPending }
+                .forEach {
+                    testCertificateRepository.markCertificateAsSeenByUser(it.identifier)
+                }
+        }
+        .map { }
```

Comment:
map { } -> { }

Reply:
What do you mean by that?
We just need to be subscribed, we don't want to return a specific value, to achieve that: `.map { Unit }` -> `.map { }` -> `Unit`, is there a better way?

Reply:
without the unit it seemed to me like a forgotten leftover

Reply:
I'll make it better in another PR :heart: